var/spack/repos/builtin/packages/selalib/package.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Selalib(CMakePackage):
+    """SeLaLib is a modular library for the kinetic and gyrokinetic simulation
+    of tokamak plasmas by the semi-lagrangian or particle-in-cell methods"""
+
+    homepage = "https://selalib.github.io/selalib"
+    url = "https://github.com/selalib/selalib"
+    git = "https://github.com/selalib/selalib"
+
+    maintainers = ["pnavaro", "freifrauvonbleifrei"]
```

Comment:
We have directives for maintainers:
```suggestion
    maintainers("pnavaro", "freifrauvonbleifrei")
```
This was done to allow dropping (or not) maintainers from base classes when #35045 is merged